pkgs/tools/misc/coreutils/default.nix

```diff
@@ -80,11 +80,11 @@ stdenv.mkDerivation rec {
 
   # Works around a bug with 8.26:
   # Makefile:3440: *** Recursive variable 'INSTALL' references itself (eventually).  Stop.
-  ${if hostPlatform == buildPlatform then null else "preInstall"} = ''
+  preInstall = optionalString (hostPlatform == buildPlatform) ''
```

Comment:
Inverted logic? Isn't it so that when you're building for host != build, then you'd like to use "install" from the buildPackages set?